### ISSUE TITLE
Rename the local variable to match the regular expression (Group 5)

### ISSUE THREAD

**Issue by JayPatel286** [Mutasimur/OpenTracks-Winter-SOEN-6431_2024_Group_5#5]
### Description
Local variable and method parameter names should comply with a naming convention. Here local variable 'value_rpm' does not follow the guidelines of naming convention.

### Screenshot
<img width="772" alt="Screenshot 2024-02-14 at 3 41 24 PM" src="https://github.com/Mutasimur/OpenTracks-Winter-SOEN-6431_2024_Group_5/assets/74642952/21c21c86-bbc5-426c-bcc3-4b0377af7cca">

### What is the potential impact?
Inconsistent naming of local variables and method parameters can lead to several issues in your code:

* **Reduced Readability**: inconsistent local variable and method parameter names make the code harder to read and understand; consequently, it is more difficult to identify the purpose of each variable, spot errors, or comprehend the logic.
* **Difficulty in Identifying Variables**: local variables and method parameters that don’t adhere to a standard naming convention are challenging to identify; thus, the coding process slows down, especially when dealing with a large codebase.
* **Increased Risk of Errors**: inconsistent or unclear local variable and method parameter names lead to misunderstandings about what the variable represents. This ambiguity leads to incorrect assumptions and, consequently, bugs in the code.
* **Collaboration Difficulties**: in a team setting, inconsistent naming conventions lead to confusion and miscommunication among team members.
* **Difficulty in Code Maintenance**: inconsistent naming leads to an inconsistent codebase. The code is difficult to understand, and making changes feels like refactoring constantly, as you face different naming methods. Ultimately, it makes the codebase harder to maintain.

In summary, not adhering to a naming convention for local variables and method parameters can lead to confusion, errors, and inefficiencies, making the code harder to read, understand, and maintain.

### How to fix it?

* Rename your local variable or method parameter to follow your project’s naming convention to address this issue.
* Follow camel case naming convention (Where every word except the first word of variable name starts with uppercase letter).

### Impact after resolution

* Reduced Technical Debt
* Enhanced Code Quality
